math/fgsl/Portfile

```diff
@@ -4,19 +4,26 @@ PortSystem          1.0
 PortGroup           compilers 1.0
 PortGroup           github 1.0
 
-github.setup        reinh-bader fgsl 1.4.0 v
+github.setup        reinh-bader fgsl 1.5.0
+github.tarball_from archive
 revision            0
 categories          math science
-maintainers         {takeshi @tenomoto} openmaintainer
+maintainers         {takeshi @tenomoto} \
+                    {@Dave-Allured noaa.gov:dave.allured} \
+                    openmaintainer
 license             GPL-2
+
 description         Fortran interface to the GNU scientific library
-long_description \
-    A portable, object-based Fortran interface to the GNU scientific library, \
-    a collection of numerical routines for scientific computing.
-homepage            https://doku.lrz.de/display/PUBLIC/FGSL+-+A+Fortran+interface+to+the+GNU+Scientific+Library
-checksums           rmd160  580a519dc98ab1a774e5b50b347de1bfe0222171 \
-                    sha256  7b1b33d7165b756279c26ecd4b27edb19ee0c11165e13bf518cd7f91019d1b39 \
-                    size    2880105
+long_description    A portable, object-based Fortran interface to the GNU scientific library \
+                    (GSL), a collection of numerical routines for scientific computing. \
+                    Version 1.5.x is for use with GSL versions >= 2.6.
+
+homepage            https://doku.lrz.de/fgsl-a-fortran-interface-to-the-gnu-scientific-library-10746505.html \
+                    https://github.com/reinh-bader/fgsl/
```

Comment:
this should have only one entry, see: https://trac.macports.org/ticket/68460 and it's already set by the `github` PG to the second one.

Reply:
Homepage:  Fixed as requested.